File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/DragElementStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;// 拖拽元素


import com.mokatest.platform.demos.domain.ui.dto.ElementDTO;
import lombok.Data;

@Data
public class DragElementStepDTO extends StepBaseDTO {
    private String stepType; // 步骤类型
    private ElementDTO dragElement; // 被拖拽元素
    private ElementDTO targetElement; // 目标元素
}

File: frontEnd/src/types/dto/stepDTO/DragElementStepDTO.ts
// 拖拽元素

import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";
import {Element} from '@/types/domain/Element';
import {ElementDTO} from "@/types/dto/ElementDTO";

export class DragElementStepDTO extends StepBaseDTO {
    stepType: string = 'DRAG'; // 步骤类型
    dragElement: ElementDTO = new ElementDTO(); // 被拖拽元素
    targetElement: ElementDTO = new ElementDTO(); // 目标元素
}